src/eko/evolution_operator/operator_matrix_element.py

```diff
@@ -49,7 +49,7 @@ def build_ome(A, matching_order, a_s, backward_method):
     # .end
     ome = np.eye(len(A[0]), dtype=np.complex_)
     A = np.ascontiguousarray(A)
-    if backward_method == "expanded":
+    if backward_method.value == "expanded":
```

Comment:
The proper way is to compare with the actual object:

```suggestion
    if backward_method is InversionMethod.EXPANDED:
```